app-misc/goobook/goobook-3.3.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+PYTHON_COMPAT=( python3_6 )
+
+inherit distutils-r1 readme.gentoo-r1
+
+DESCRIPTION="Access your Google contacts from the command line"
+HOMEPAGE="https://gitlab.com/goobook/goobook"
+SRC_URI="mirror://pypi/g/${PN}/${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+RDEPEND="${PYTHON_DEPS}
+	>=dev-python/google-api-python-client-1.6.4[${PYTHON_USEDEP}]"
+# dev-python/{simplejson,oauth2client} are deps for the above
+
+DEPEND="${PYTHON_DEPS}
+	dev-python/setuptools[${PYTHON_USEDEP}]"
+
+src_install() {
+	distutils-r1_src_install
+	readme.gentoo_create_doc
+}
```

Comment:
Move `src_install()` to be in between `DOC_CONTENTS="..."` and `pkg_postinst()`.